### PR TITLE
Remove trailing slash

### PR DESCRIPTION
**Pull request by atukai**
It introduce 404 for me

### CODE REVIEW

File: config/module.config.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
                     'display'   => [
                         'type' => 'Segment',
                         'options' => [
-                            'route'     => '/display/:filter/',
```
Comment (ojhaujjwal) [inline config/module.config.php]: This can introduce a BC break. How about `[/]`?

**Comment by atukai** [comment]
I found that 404 happened, because i remove trailing slash with nginx. [/] will be a good solution for me.

**Comment by ojhaujjwal** [comment]
Can you update the PR to use `[/]`? I will merge it.

**Comment by ojhaujjwal** [comment]
@atukai Merged. Thanks!

**Pull request merged**